cookbooks/aws-parallelcluster-config/recipes/directory_service.rb

```diff
@@ -49,6 +49,24 @@
     command "cp #{sssd_conf_path} #{shared_sssd_conf_path}"
     sensitive true
   end
+
+  if node['cluster']["directory_service"]["generate_ssh_keys_for_users"] == 'true'
+    generate_ssh_key_path = "#{node['cluster']['scripts_dir']}/generate_ssh_key.sh"
+    template generate_ssh_key_path do
+      source 'directory_service/generate_ssh_key.sh.erb'
+      owner 'root'
+      group 'root'
+      mode '0600'
+    end
+    filter_lines 'Configure PAM postlogin script to call generate SSH key script' do
+      path generate_ssh_key_path
+      filters(
+        [
+          { before: [/^session.*/, "session     optional      pam_exec.so #{generate_ssh_key_path}", :first] },
```

Comment:
should we log this to something in /var/log/parallelcluster? maybe something like /var/log/parallelcluster/pam_ssh_key_generator.log?

Reply:
Done